x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -1330,4 +1330,69 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'mac.advanced.kernel.fileaccess',
+    first_supported_version: '8.11',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.kernel.fileaccess',
+      {
+        defaultMessage: 'Sends file access events to the stack. Default: false',
+      }
+    ),
+  },
+  {
+    key: 'mac.advanced.events.image_load',
+    first_supported_version: '8.11',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.mac.advanced.events.image_load',
+      {
+        defaultMessage:
+          'Collects image load events and use them for behaviour protection. By default, does not send events to the stack. Default: true',
```

Comment:
Do any main policy checkboxes control `fileaccess` and `image_load`? If yes, then these options should default to abiding by the main checkbox, and only be the "force off" options. If no, then why not?

Reply:
`fileaccess` is controlled by the file checkbox. If files events are capture, access events will be as well, but they will not be sent to ES unless fileaccess forces it.

`mac.events.image_load` should be a new checkbox that enables image load events to be collected (I think it was a mistake, it shouldn't be under the `advanced` group).

`mac.advanced.image_load.capture` is used to force dylib events to be sent to the stack.